packages/uniforms-bootstrap3/src/RadioField.tsx

```diff
@@ -14,38 +15,37 @@ export type RadioFieldProps = HTMLFieldProps<
   string,
   HTMLDivElement,
   {
-    allowedValues?: string[];
+    options?: Option<string>[];
     inline?: boolean;
     inputClassName?: string;
-    transform?: (value: string) => string;
   }
 >;
 
 function Radio(props: RadioFieldProps) {
   return wrapField(
     props,
-    props.allowedValues?.map(item => (
+    props.options?.map(item => (
       <div
-        key={item}
+        key={item.key ?? item.value}
         className={classnames(
           props.inputClassName,
           `radio${props.inline ? '-inline' : ''}`,
         )}
       >
-        <label htmlFor={`${props.id}-${escape(item)}`}>
+        <label htmlFor={`${props.id}-${escape(item.key ?? item.value)}`}>
```

Comment:
There's no need to `escape` the `key`:
```suggestion
        <label htmlFor={`${props.id}-${item.key ?? escape(item.value)}`}>
```